db/db_block_cache_test.cc

```diff
@@ -377,8 +380,8 @@ TEST_F(DBBlockCacheTest, IndexAndFilterBlocksStats) {
             index_bytes_insert);
   ASSERT_GT(TestGetTickerCount(options, BLOCK_CACHE_FILTER_BYTES_INSERT),
             filter_bytes_insert);
-  ASSERT_EQ(TestGetTickerCount(options, BLOCK_CACHE_INDEX_BYTES_EVICT),
-            index_bytes_insert);
+  // ASSERT_EQ(TestGetTickerCount(options, BLOCK_CACHE_INDEX_BYTES_EVICT),
+  //           index_bytes_insert);
```

Comment:
Is this assert intended to be removed permanently or only temporarily? 
Depending on the answer you can either delete it or add a comment to explain

Reply:
Just temporarily, similarly to the previous one. I'll add the same comment here.